src/sentry/web/frontend/accept_organization_invite.py

```diff
@@ -74,6 +74,7 @@ def handle(self, request, member_id, token):
             # Show login or register form
             request.session['_next'] = request.get_full_path()
             request.session['can_register'] = True
+            request.session['invite_email'] = om.email
```

Comment:
it'd be good to pop this value out of the session at some point (ideally when can_register is cleared)